packages/cli/src/utils.ts

```diff
@@ -219,6 +234,27 @@ export function writeYaml(data: any, filename: string, noRefs = false) {
   fs.writeFileSync(filename, content);
 }
 
+export function writeJson(data: unknown, filename: string) {
+  const content = JSON.stringify(data, null, 2);
+
+  if (process.env.NODE_ENV === 'test') {
+    process.stderr.write(content);
+    return;
+  }
+  fs.mkdirSync(dirname(filename), { recursive: true });
+  fs.writeFileSync(filename, content);
+}
+
+export function getAndValidateFileExtension(fileName: string): string {
```

Comment:
I wonder if it makes sense to be more precise here:

```suggestion
export function getAndValidateFileExtension(fileName: string): 'yaml' | 'yml' | 'json' {
```

Reply:
Make sense, Thank you